packages/git/src/node/dugite-git.spec.ts

```diff
@@ -342,25 +342,54 @@ describe('git', async function () {
 
 });
 
-async function createGit(fsRoot: string = ''): Promise<DugiteGit> {
+async function createGit(): Promise<DugiteGit> {
     const container = new Container();
     const bind = container.bind.bind(container);
     bindLogger(bind);
     container.rebind(ILoggerServer).to(ConsoleLoggerServer).inSingletonScope();
-    bindGit(bind);
+    bindGit(bind, {
+        bindManager(binding: interfaces.BindingToSyntax<{}>): interfaces.BindingWhenOnSyntax<{}> {
+            return binding.to(TestGitRepositoryManager).inSingletonScope();
+        },
+        bindWatcher(binding: interfaces.BindingToSyntax<{}>): interfaces.BindingWhenOnSyntax<{}> {
+            return binding.to(TestRepositoryWatcher);
+        }
+    });
     return container.get(DugiteGit);
 }
 
-async function createWorkspace(fsRoot: string): Promise<WorkspaceServer> {
-    return {
+@injectable()
+class TestGitRepositoryManager implements GitRepositoryManager {
```

Comment:
Could you move these mocks under node/test/ClassName ?

So that they are reusable? 

Like do in core/src/common/test/mock-logger.ts ?

Reply:
:+1: Sure.